### PR TITLE
add more entropy-like cones

### PR DESCRIPTION
**Pull request by lkapelevich**
TODOs
- [x] finish correctors math
- [x] move shared functions into Cones.jl
- [ ] simplifications, allocations etc.
- [ ] inverse Hessians
- [x] ~think about separating symm_kron for symmetric mat~ don't see a reason to at this point
- [ ] hess prods

### CODE REVIEW

File: src/Cones/epipertraceentropytri.jl [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    cone.feas_updated = true
+    return cone.is_feas
+end
+
+function is_dual_feas(cone::EpiPerTraceEntropyTri{T}) where T
+    u = cone.dual_point[1]
+    v = cone.dual_point[2]
+    @views W = Hermitian(svec_to_smat!(similar(cone.point, cone.d, cone.d), cone.dual_point[3:cone.dim], cone.rt2), :U)
+    if u > eps(T)
+        return v - u * tr(exp.(-W ./ u - I)) > eps(T)
```
Comment (chriscoey) [inline src/Cones/epipertraceentropytri.jl:115]: i think there's a mistake here

Comment (chriscoey) [reply src/Cones/epipertraceentropytri.jl:115]: maybe `tr(exp(-W / u - I))` (remove the dots)

Comment (lkapelevich) [reply src/Cones/epipertraceentropytri.jl:115]: This should be simplified. But there are weird things happening in BigFloat.

Comment (chriscoey) [reply src/Cones/epipertraceentropytri.jl:115]: `tr(exp.(-W ./ u - I)) = sum(exp(-W[i,i] / u - 1) for i in 1:size(W,1))`
it ignores all but the diagonal elements of W

Comment (lkapelevich) [reply src/Cones/epipertraceentropytri.jl:115]: oops yes the dot was a mistake. anyway i'll push a corrected function soon.

Comment (lkapelevich) [reply src/Cones/epipertraceentropytri.jl:115]: there were other things wrong

**Pull request merged**